configs/config.clevo_ns70pu

```diff
@@ -22,3 +24,13 @@ CONFIG_I2C_TRANSFER_TIMEOUT_US=500000
 CONFIG_SMMSTORE=y
 CONFIG_SMMSTORE_V2=y
 CONFIG_SMMSTORE_SIZE=0x40000
+# CONFIG_TPM_PPI is not set
+CONFIG_PAYLOAD_EDK2=y
+CONFIG_EDK2_REPOSITORY="https://github.com/Dasharo/edk2"
+CONFIG_EDK2_TAG_OR_REV="abfdef4027579238d19501973ef8f3add1d561ce"
+CONFIG_EDK2_SD_MMC_TIMEOUT=10
+CONFIG_EDK2_BOOT_MENU_KEY=0x0016
+CONFIG_EDK2_SETUP_MENU_KEY=0x0017
+CONFIG_EDK2_BOOTSPLASH_FILE="3rdparty/dasharo-blobs/novacustom/bootsplash.bmp"
+CONFIG_VBOOT=y
+CONFIG_VBOOT_MOCK_SECDATA=y
```

Comment:
Can be moved to the board's Kconfig. Please also see how it's done in TGL based NS5x, the while Vboot section can probably be copied.

Reply:
Actually this is completely unnecessary since we're using a discrete TPM.

Reply:
Done